port/raspberrypi/rp2xxx/src/hal.zig

```diff
@@ -22,6 +22,7 @@ pub const rtc = switch (compatibility.chip) {
 };
 pub const spi = @import("hal/spi.zig");
 pub const i2c = @import("hal/i2c.zig");
+pub const i2c_slave = @import("hal/i2c_slave.zig");
```

Comment:
Maybe this should be exported through i2c.zig, e.g. have i2c.zig import it and them have a `pub slave`

Reply:
Done!